src/vte.c

```diff
@@ -221,6 +224,78 @@ WRAP_RGBA_SETTER(vte_terminal_set_color_foreground)
 #endif
 
 
+#if GTK_CHECK_VERSION(3, 0, 0)
+static void default_vte_terminal_set_background_image_file(VteTerminal *terminal, const char *path)
+{
+	if (background_image_pixbuf)
+	{
+		g_object_unref(background_image_pixbuf);
+		background_image_pixbuf = NULL;
+	}
+
+	if (path != NULL)
```

Comment:
Should allow for an empty `path` as none, as the setting will be the empty string rather than NULL, so this will lead to try and open file `""` and lead to a warning.

BTW, apparently VTE's implementation of this fails silently when the file can't be open/open as an image.  Warning is better for us I suppose, though.